src/main/java/com/cloudbees/jenkins/support/impl/LoadStats.java

```diff
@@ -0,0 +1,155 @@
+package com.cloudbees.jenkins.support.impl;
+
+import com.cloudbees.jenkins.support.api.Component;
+import com.cloudbees.jenkins.support.api.Container;
+import com.cloudbees.jenkins.support.api.Content;
+import com.cloudbees.jenkins.support.api.PrintedContent;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.model.Label;
+import hudson.model.LoadStatistics;
+import hudson.model.MultiStageTimeSeries;
+import hudson.model.TimeSeries;
+import hudson.security.Permission;
+import jenkins.model.Jenkins;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TimeZone;
+import java.util.TreeMap;
+
+/**
+ * @since 
+ */
+@Extension
+public class LoadStats extends Component {
+    @NonNull
+    @Override
+    public Set<Permission> getRequiredPermissions() {
+        return Collections.emptySet();
+    }
+
+    @NonNull
+    @Override
+    public String getDisplayName() {
+        return "Load Statistics";
+    }
+
+    @Override
+    public void addContents(@NonNull Container container) {
+        Jenkins jenkins = Jenkins.getInstance();
+        add(container, "unlabeled", jenkins.unlabeledLoad);
+        add(container, "overall", jenkins.overallLoad);
+        for (Label l: jenkins.getLabels()) {
+            try {
+                add(container, String.format("label/%s", URLEncoder.encode(l.getName(), "UTF-8")), l.loadStatistics);
+            } catch (UnsupportedEncodingException e) {
+                // ignore UTF-8 is required by JLS specification
+            }
+        }
+    }
+
+    private void add(@NonNull Container container, String name, LoadStatistics stats) {
+        boolean headless = Boolean.getBoolean("java.awt.headless");
```

Comment:
this is the wrong way to check for a headless enviroment - you should check the GraphicsEnvironment.isHeadless()